transit_odp/pipelines/pipelines/dataset_etl/transform.py

```diff
@@ -44,14 +48,33 @@ def transform(self, extracted_data: ExtractedData) -> TransformedData:
         jp_sections = extracted_data.jp_sections.copy()
         timing_links = extracted_data.timing_links.copy()
         stop_points = extracted_data.stop_points.copy()
+        vehicle_journeys = self.extracted_data.vehicle_journeys.copy()
         provisional_stops = extracted_data.provisional_stops.copy()
+        flexible_stop_points = extracted_data.flexible_stop_points.copy()
+        flexible_journey_patterns = extracted_data.flexible_journey_patterns.copy()
+        flexible_journey_details = extracted_data.flexible_journey_details.copy()
 
         # Match stop_points with DB
         stop_points = self.sync_stop_points(stop_points, provisional_stops)
         stop_points = sync_localities_and_adminareas(stop_points)
         # stop_points = self.sync_admin_areas(stop_points)
         most_common_localities = get_most_common_localities(stop_points)
 
+        flexible_stops = stop_points.loc[
+            stop_points.index.isin(flexible_stop_points.index)
+        ]
+        flexible_stop_points = flexible_stops.merge(
+            flexible_stop_points, left_index=True, right_index=True
+        )
+        stop_points = stop_points.loc[
+            ~stop_points.index.isin(flexible_stop_points.index)
+        ]
+
+        if not vehicle_journeys.empty and not journey_patterns.empty:
```

Comment:
Not sure why this is showing up as newly added lines while its already in dev

Reply:
this file is not being used its the duplicated code we need to update for the test cases